content/analyst/sfmc/auto-seeding-with-salesforce-marketing-cloud-inbox-tracker-app.md

```diff
@@ -0,0 +1,48 @@
+---
+lastUpdated: "02/23/2023"
+title: Auto-Seeding With Salesforce Marketing Cloud Inbox Tracker App
+description: This article will provide you with an overview of the seeding capability available to Salesforce Marketing Cloud Inbox Tracker App users
+---
+
+## The Inbox Tracker Salesforce Marketing Cloud application allows for auto and ad-hoc seeding through Email Studio or Automation Studio.
+
+ In this guide, we'll go over:
+
+* **[What are IntelliSeeds™?](#what-are-intelliseeds-)**
+* **[How to Configure Auto-Seeding](#how-to-configure-auto-seeding)**
+* **[Seeding Options via Email and Automation Studio](#seeding-options-via-email-and-automation-studio)**
+
+### What are IntelliSeeds™?
+
+ Our Salesforce Marketing Cloud seeding integration supports sends to IntelliSeeds™, or virtual seeds.
+
+ IntelliSeeds™ virtual users who have behavior attributes modeled after real subscribers from our panel. They automatically subscribe themselves to email lists, read and delete at a certain cadence, and even go on vacation. Since IntelliSeeds™ look like a regular subscriber on your list to an ISP, having your sends to the virtual seeds to match your regular subscribers will result in the closest supplemental datapoint to panel available.
+
+### How to Configure Auto-Seeding
+
+ Within the Salesforce Marketing Cloud Inbox Tracker App, auto seeding can be configured on a business unit level by selecting *Settings* *Seed List Integration*.
+
+![](media/auto_seeding_with_salesforce_marketing_cloud_inbox_tracker_app/image_0.png)
+
+ Within the Inbox Tracker UI, you can configure custom sets of IntelliSeed™ lists by selecting the *Settings Cog*  *Configure/Download IntelliSeed™ List.* Please read [this article](/analyst/inbox-and-design-tracker/how-to-deploy-intelliseed-and-seed-lists#h_495ecb3d2f) for best practice guidance to consider when making these groups.
```

Comment:
`/analyst/inbox-and-design-tracker/how-to-deploy-intelliseed-and-seed-lists#h_495ecb3d2f` -> `/analyst/inbox-and-design-tracker/intelliseed-sending-guide/`

Reply:
Bah, thought I got this one